### ISSUE TITLE
Perform a check whether xcode command line tool is installed

### ISSUE THREAD

**Issue by DiksonH** [Floris497/mac-pixel-clock-patch-V2#101]
It fails on me on the first time with a xcrun error (xcode not installed). But the rest of the code continues to run and I have to move the backup in recovery mode as a result.
Is it possible to check if xcode command line tool is installed to prevent this hassle?

**Comment by Floris497** [comment]
Yes certainly possible, I'll make a check and see how to trigger the installation prompt.

**Comment by croaking** [comment]
Honestly the whole thing could simply be done in a shell script, check for the version of OS X, then make a backup of the files (actually 2 backups would be best one to a .orig filename that once that file exists it’s never touched again and cannot be overwritten chmod -w will take care of that).  Then it would check for SIP and then do the appropriate work to apply the fix.  No need to have 4 or 5 separate files.

**Comment by Floris497** [comment]
4 or 5 separate files? I'm not really sure what you are aiming for?

**Comment by croaking** [comment]
What I’m saying is that right now you have a zip file with 4 files, there’s no reason for that, remove the ambiguity of which one to use by just putting them together, use some if/then or case logic to determine which patch to use, and people will stop messing up and using the wrong version.

XXXX-MacBook:~ username$ sw_vers
ProductName:	Mac OS X
ProductVersion:	10.12.1
BuildVersion:	16B2657

XXXX-MacBook:~ username$

and so-forth…

**Comment by Floris497** [comment]
It's not that simple.. Nvidia for who wants Nvidia IOKit for certain Nvidia and CoreDisplay for Intel GPU. This is the easiest way.. I could put 3 download links in the top of the readme to make it easier to download a single file.. I have good reasons to do it this way.. there's too much unnecessary code needed to make it work easy.
